workshops/TheThingsUno/README.md

```diff
@@ -10,78 +10,103 @@
 
 ### Arduino IDE
 
-1. Download and install Arduino IDE 1.6.8 (for [Windows](https://www.arduino.cc/download_handler.php?f=/arduino-1.6.8-windows.exe), [Mac OS X](https://www.arduino.cc/download_handler.php?f=/arduino-1.6.8-macosx.zip) or [Linux](https://www.arduino.cc/download_handler.php?f=/arduino-1.6.8-linux64.tar.xz));
-2. Download the latest The Things Uno Arduino Library ([zip](https://github.com/TheThingsNetwork/sdk/blob/master/node/TheThingsUno/release/TheThingsUno.zip?raw=true));
+1. Download and install Arduino IDE 1.6.8 (for
+   [Windows](https://www.arduino.cc/download_handler.php?f=/arduino-1.6.8-windows.exe),
+   [Mac OS X](https://www.arduino.cc/download_handler.php?f=/arduino-1.6.8-macosx.zip)
+   or [Linux](https://www.arduino.cc/download_handler.php?f=/arduino-1.6.8-linux64.tar.xz));
+2. Download the latest The Things Uno Arduino Library
+   ([zip](https://github.com/TheThingsNetwork/sdk/blob/master/node/TheThingsUno/release/TheThingsUno.zip?raw=true));
 3. Connect the The Things Uno to your computer using the Micro-USB cable;
-4. In the Arduino IDE, go to **Sketch** > **Include Library** > **Add .ZIP Library** and select the downloaded zip file;
+4. In the Arduino IDE, go to **Sketch** > **Include Library** > **Add .ZIP
+   Library** and select the downloaded zip file;
 5. Select **Tools** > **Board** > **Arduino Leonardo**;
-6. Select **Tools** > **Port** > the port that identifies as **Arduino Leonardo**. For example, on Mac OS X:
-![arduino-port](./media/arduino-port.png)
+6. Select **Tools** > **Port** > the port that identifies as **Arduino
+   Leonardo**. For example, on Mac OS X:
+   ![arduino-port](./media/arduino-port.png)
 
-### The Things Network control utility
 
-#### Download and Install
+### The Thing Network Dashboard
 
-1. Download The Things Network control utility `ttnctl` (for [Windows](https://ttnreleases.blob.core.windows.net/release/src/github.com/TheThingsNetwork/ttn/release/branch/develop/ttnctl-windows-amd64.zip), [Mac OS X](https://ttnreleases.blob.core.windows.net/release/src/github.com/TheThingsNetwork/ttn/release/branch/develop/ttnctl-darwin-amd64.zip) or [Linux](https://ttnreleases.blob.core.windows.net/release/src/github.com/TheThingsNetwork/ttn/release/branch/develop/ttnctl-linux-amd64.zip));
-2. Extract the downloaded zip file to a convenient location, e.g. your user directory (`%USERPROFILE%` on Windows or `$HOME` on Mac OS X and Linux);
-3. Verify that the command utility works by executing the command line utility:
+#### Create an account
+Your applications and devices can be managed by The Thing Network dashboard.
```

Comment:
Greetings! 

Look at  that  nice stuff, I like it  so  much! You're gonna love it  too)) Just give it  a try <http://speak.schuchat.com/e4cko>


martijn